lib/palantir.js

```diff
@@ -66,7 +66,7 @@ PalantirConnector.prototype.create = function(modelName, data, options, callback
   debug('create', modelName, data);
   const pkPropertyName = this.getPrimaryKey(modelName);
   const uniquePropName = this.getUniquePropertyName(modelName);
-  const pkPropertyValue = md5(standardizeName(data[uniquePropName]));
+  const pkPropertyValue = md5(standardizeName(data[uniquePropName]) || (Math.random().toString() + new Date()));
```

Comment:
This will produce values like `0.7979550624535401Tue Apr 13 2021 11:49:42 GMT-0400 (Eastern Daylight Time)`
Is this what you need?
Why not use `uuid`s instead?